mbuild/packing.py

```diff
@@ -843,7 +926,11 @@ def _run_packmol(input_text, filled_xyz, temp_file):
             "sufficient packing result."
         )
         warnings.warn(msg)
+<<<<<<< HEAD
+        os.system('cp {0}_FORCED {0}'.format(filled_xyz.name))
+=======
         os.system("cp {0}_FORCED {0}".format(filled_xyz.name))
+>>>>>>> 636f9b15f796614c616f807c652486010fa9e43e
```

Comment:
```suggestion
        os.system("cp {0}_FORCED {0}".format(filled_xyz.name))
```
oops I think we committed a merge conflict